.travis.yml

```diff
@@ -37,7 +41,16 @@ before_script:
 
   # Set up Go environment for build.
   - export GOPATH=$HOME/.gopath
+
   # Download godep tool and make add it to PATH environment variable.
   - go get github.com/tools/godep
   - export PATH=$PATH:$GOPATH/bin
+
+  # Download etcd and make add it to PATH environment variable.
+  - curl -fsSL https://github.com/coreos/etcd/releases/download/v2.2.1/etcd-v2.2.1-linux-amd64.tar.gz | tar xzf -
+  - ln -fns etcd-v2.2.1-linux-amd64 etcd
+  - export PATH=$PWD/etcd:$PATH
+
+  # Pull golang image
```

Comment:
This docker golang image is used by local-up-cluster.sh script to build kubernetes. During build process script ask if you want pull golang image. There is no way to type 'yes' so it must be install in advance otherwise compilation process will hang up.